style.css

```diff
@@ -98,3 +103,241 @@ a {
 p a {
   color: #7f8cff;
 }
+
+/* signig worection */
+.workcards {
+  display: grid;
+  row-gap: 2em;
+  padding: 100px 25px;
+}
+
+.card {
+  background-color: #fff;
+  padding: 20px;
+  border-radius: 20px;
+  min-height: 80vh;
+  display: flex;
+  flex-direction: column;
+}
+
+.pic1 img {
+  width: 100%;
+}
+
+.title1 {
+  font-family: Poppins, sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  letter-spacing: 0;
+}
+
+.frame-container {
+  display: flex;
+  flex-direction: row;
+  text-decoration: none;
+  margin-top: -12px;
+  font-family: Poppins, sans-serif;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0;
+  padding: 0%;
+  margin-left: -5px;
+}
+
+.role,
+.year {
+  color: #7a869a;
+}
+
+.card-text {
+  font-family: Poppins, sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.language-container {
+  display: flex;
+  flex-direction: row;
+  padding: 0;
+  margin-top: 0;
+}
+
+.language {
+  font-family: Poppins, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  text-align: left;
+}
+
+.html,
+.css,
+.java {
+  background-color: #ebebff;
+  color: #6070ff;
+  border-radius: 12px;
+  padding: 10px;
+}
+
+.card-btn {
+  text-align: left;
+  background-color: white;
+  color: #396df2;
+  width: fit-content;
+  border-radius: 5px;
+  font-family: Poppins, sans-serif;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  border: 1px solid #396df2;
+  padding: 10px;
+}
+
+.card-btn:hover {
+  background-color: #396df2;
+  color: white;
+}
+
+.title {
+  width: 34%;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.bottom {
+  margin: 0 0 16px -25px;
+}
+
+.bio {
+  font-family: Poppins, sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #344563;
+  padding: 0 20px;
+  margin-left: 20px;
+}
+
+.contact-container {
+  display: flex;
+  flex-direction: column;
+}
+
```

Comment:
- [ ] Kindly make sure to use the given background color for the about section, which is white ✨ After applying a different background color for testing, I noticed the the space/padding/margin is not according to the given design guidelines. See the images below for more clarity 💡. Kindly fix the following highlighted areas according to the given design Also, kindly apply the border radius. 

Your design | Given design
---| ---|
![image](https://github.com/ali-techworld/Portfolio-Setup/assets/106119105/65545206-f2e0-4245-915b-3da9e5f65dd5) | ![image](https://github.com/ali-techworld/Portfolio-Setup/assets/106119105/adf4ae5d-d605-46ae-8732-573242a22126)